integration/test/ParseQueryTest.js

```diff
@@ -1223,6 +1223,18 @@ describe('Parse Query', () => {
     });
   });
 
+  it('can return all objects with findAll', async () => {
```

Comment:
Looks good! Can you add unit tests to increase coverage?

Reply:
I've added some unit tests. I basically took the ones from .eachBatch but using the findAll method. I hope it's enough.

Reply:
Looks good!